Native/CommandPaletteNucleoFFI/src/lib.rs

```
@@ -482,6 +495,59 @@ fn keyword_exact_line_score(query_char_count: usize) -> f64 {
 
+fn title_phrase_score(
+    candidate: &Candidate,
+    query_lowercase: &str,
+    query_token_count: usize,
+) -> Option<f64> {
+    if query_lowercase.is_empty() {
+        return None;
+    }
+
+    let title = &candidate.title_lowercase;
+    let Some(start_byte) = title.find(query_lowercase) else {
```

Clippy (1.95.0)

[warning] this `let...else` may be rewritten with the `?` operator

(warning)